recipes/bison/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import glob
+import os
+import shutil
+
+
+class ConanFileDefault(ConanFile):
+    name = "bison"
+    url = "https://github.com/conan-io/conan-center-index   "
+    homepage = "https://www.gnu.org/software/bison/"
+    description = "Bison is a general-purpose parser generator"
+    topics = ("conan", "bison", "parser")
+    license = "GPL-3.0-or-later"
+    exports_sources = ["patches/*.patch"]
+    _source_subfolder = "source_subfolder"
+    requires = ("m4/1.4.18", "flex/2.6.4")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
```

Comment:
Doesn't it have a shared option?

Reply:
I tried it, but it always generates a static lib